ui/src/app/components/object-detail/object-detail.component.scss

```diff
@@ -62,6 +62,17 @@
   }
 }
 
+.table {
+  height: 100%;
+  background-color: white;
+  .mat-mdc-tab-group {
+    height: 93%;
```

Comment:
I tested and tweaked with these numbers to make sure sequence table and other tables show properly

Reply:
So the concern here is that depending on screen sizes and resolutions this stuff changes.
So it is better to use a standard CSS for all the elements dependent on type, so that these changes don't go completely random on different screen sizes.

Reply:
Looking back, I think I made a blunder. These values are already set, but the values are same. Should we modify them or should that be a separate effort?:
https://github.com/GoogleCloudPlatform/spanner-migration-tool/blob/master/ui/src/app/components/object-detail/object-detail.component.scss#L54-L63